internal/features/stacks/events.go

```diff
@@ -205,34 +205,38 @@ func (f *StacksFeature) decodeStack(ctx context.Context, dir document.DirHandle,
 			}
 
 			spawnedIds, err := loadStackComponentSources(ctx, f.store, f.bus, path)
+			if err != nil {
+				return spawnedIds, err
```

Comment:
Do we need to return here? I think we can still try to load the schemas, even if decoding the components fails

Reply:
Would you then just ignore the error or keep it for later? Which error would you return if loading the schemas returns one as well?

Reply:
I would log the error from `loadStackComponentSources` and proceed with loading the schemas.

If scheduling the schema job fails, something must really be broken, so I think returning there makes sense.